semeio/workflows/localisation/local_script_lib.py

```diff
@@ -744,6 +777,155 @@ def add_ministeps(
         ert_local_config.getUpdatestep().attachMinistep(ministep)
 
 
+def get_corr_group_spec(correlations_spec_list, name):
+    found = False
+    corr_spec = None
+    for count, corr_spec in enumerate(correlations_spec_list):
+        corr_group_name = corr_spec.name
+        if name == corr_group_name:
+            found = True
+            break
+    if not found:
+        raise ValueError(
+            f"Can not find correlation group: {name} in user specification."
+        )
+    return corr_spec
+
+
+def verify_ministep_active_param(
```

Comment:
Thought this could be removed?

Reply:
They are used in test_integration.py   I will move them to that file? They are used in check_consistency_for_active_param_and_obs in test_integration.py